air/src/execution_step/execution_context/streams_variables.rs

```diff
@@ -21,6 +21,7 @@ mod utils;
 use crate::execution_step::ExecutionResult;
 use crate::execution_step::Stream;
 use crate::ExecutionError;
+use air_interpreter_data::GenerationIdx;
```

Comment:
It should be arranged according to these rules https://github.com/fluencelabs/aquavm/blob/master/docs/developer-notes.md